### DISCUSSION TITLE
Create switches as stack

### DISCUSSION THREAD

**Discussion by sadden3194** [netbox-community/netbox-docker#1019] [General]
Hi all,

I need support.
I have 3 switches which are configured as a stack.
But I dont unterstand how can I create it in netbox and placed it into a rack.

can anyone provide me a step by step guide?

**Comment by sadden3194** [comment]
nobody??

**Reply by fatman00** [reply]
You need to look at virtual chassis in netbox. create 3 physical switches and then add them to the same Virtual Chassis.
